src/components/ApiDocs/display/ApiModalBreadcrumbs.tsx

```diff
@@ -0,0 +1,55 @@
+import { useEffect, useRef, Fragment } from 'react';
+import { Text, Flex } from '@aws-amplify/ui-react';
+
+import { TypeLink, TypeLinkInterface } from './TypeLink';
+
+interface ApiModalBreadcrumbs {
+  items?: TypeLinkInterface[];
+}
+
+export const ApiModalBreadcrumbs = ({ items }: ApiModalBreadcrumbs) => {
+  const navRef = useRef<HTMLElement>(null);
+
+  useEffect(() => {
+    if (navRef.current) {
+      navRef.current.scrollLeft = navRef.current.scrollWidth;
+    }
+  }, [items]);
+
+  return (
+    <Flex
+      as="nav"
+      ref={navRef}
+      aria-label="API Type breadcrumbs"
+      className="api-modal__breadcrumbs"
+      tabIndex={0}
+    >
+      {items
+        ? items.map((item, index) => {
+            return (
+              <Fragment key={`api-breadcrumb-${index}`}>
+                {index !== 0 ? (
+                  <div
+                    className="api-modal__breadcrumbs__separator"
+                    aria-hidden="true"
+                  >
+                    /
+                  </div>
+                ) : null}{' '}
+                {index === items.length - 1 ? (
+                  <Text as="span" className="api-modal__breadcrumbs__current">
+                    {item.linkData.name}
+                  </Text>
```

Comment:
nit: should there be a space character here too? I think in the staging site the last breadcrumb is right next to the forward slash. (Not sure if this is the right spot in the code though)

<img width="822" alt="image" src="https://github.com/user-attachments/assets/4f87e118-0b4b-412f-baab-5d586b77b20b">